tsconfig.app.json

```diff
@@ -9,7 +9,7 @@
     "src/polyfills.ts"
   ],
   "include": [
-    "src/**/*.ts"
+    "src/**/*.d.ts"
```

Comment:
Why is this changed?

Reply:
I honestly can't remember doing this :/ must be unintentional -reverting back..